### PR TITLE
News stuff

### PR DESCRIPTION
**Pull request by enginnerd**
Activity transition animation, relative time stamp, and made visible the count label for discover.

@mikeng13 or @styliii can you take a look at the relative time stamp method, I think it's buggy.

### CODE REVIEW

File: WealthNOW/app/src/main/java/com/finnerds/wealthnow/helpers/DateHelper.java [hunk 1]
```diff
@@ -54,4 +56,34 @@ public static String niceDateToString(Date date) {
         sf.setLenient(true);
         return sf.format(date);
     }
+
+    public static String getRelativeTimeAgo(Date date) {
```
Comment (mikeng13) [inline WealthNOW/app/src/main/java/com/finnerds/wealthnow/helpers/DateHelper.java]: DateUtils has a relative timestamp method. Any reason we aren't using that instead? I think it's in one of the cliffnotes.

**Comment by enginnerd** [comment]
We are using it, but I added some additional features on top of it to shrink the text it returns.  I am seeing the same relative time for all news items.  Maybe the date format needs a change?

**Comment by mikeng13** [comment]
Ah yeah sorry I didn't look too closely.

**Comment by enginnerd** [comment]
ok, I finally fixed it, needed to user a different time/date format string, also styled and added it to the right of the item list.  I'll make a video in about two hours if someone can review this in the meantime. @mikeng13 @styliii

**Comment by mikeng13** [comment]
Will do! Can you make the video with the market closed though? Remember to uncomment the line where we shuffle here: https://github.com/FinNerds/AndroidApp/blob/master/WealthNOW/app/src/main/java/com/finnerds/wealthnow/fragments/AlertsFragment.java#L142

File: WealthNOW/app/src/main/java/com/finnerds/wealthnow/helpers/DateHelper.java [hunk 2]
```diff
@@ -54,4 +56,34 @@ public static String niceDateToString(Date date) {
...
+        String timeFormat = "EEE MMM dd HH:mm:ss ZZZZZ yyyy";
```
Comment (mikeng13) [inline WealthNOW/app/src/main/java/com/finnerds/wealthnow/helpers/DateHelper.java]: Let's move this into a constant like the ones we have up top.

**Comment by mikeng13** [comment]
Looks good! Just a minor dumb nitpick from me..

**Comment by enginnerd** [comment]
Ok, cool i'll merge and fix the constant, I can make a build with the market closed, and then on monday we can make a real one for a16z with live data

**Pull request merged**